File: cifrado/web/codigo/Cesarc/Cesarc/src/TransformarInverso.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Cesarc.src
{
    public class TransformarInverso
    {
        public string inverso(string texto, string texto_inverso)
        {
            for (int i = texto.Length - 1; i >= 0; i--)
                texto_inverso += texto[i];
            return texto_inverso;
        }
    }
}

File: cifrado/escritorio/codigo/Cesar/Cesar/TransformarInverso.cpp
#include "TransformarInverso.h"


string TransformarInverso::inverso(string texto, string texto_inverso)
{
	for (int i = texto.size() - 1; i >= 0; i--)
		texto_inverso += texto[i];
	return texto_inverso;
}
